docs/import/add_asset.js

```diff
@@ -425,3 +425,9 @@ function addFileToDelete(file) {
     filesToDelete.length = 0;
   }
 }
+
+$(() => {
```

Comment:
nit: I think this and the next lambda can both be one-lined :) Here and below. Maybe also extract to a function?